sh/tests/solutions/my-ls.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+alias my-ls="ls -lgSsa"
```

Comment:
with this solution the alias is never created in the student machine

Reply:
not required, the alias will be created in the working env, and its enough for the testing
but i add a part in the hints section to teach the student how to create a permanently alias